source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/Dtos/ChargeCommands/Validation/InputValidation/Factories/ChargeOperationInputValidationRulesFactory.cs

```diff
@@ -99,6 +99,8 @@ private List<IValidationRuleContainer> CreateRulesForChargeInformation(ChargeOpe
                 CreateRuleContainer(new TransparentInvoicingIsRequiredValidationRule(chargeOperationDto), chargeOperationDto.Id),
                 CreateRuleContainer(new TaxIndicatorIsRequiredValidationRule(chargeOperationDto), chargeOperationDto.Id),
                 CreateRuleContainer(new TerminationDateMustMatchEffectiveDateValidationRule(chargeOperationDto), chargeOperationDto.Id),
+                CreateRuleContainer(new TaxIndicatorIsNotAllowedForFeeValidationRule(chargeOperationDto), chargeOperationDto.Id),
+                CreateRuleContainer(new TaxIndicatorIsNotAllowedForSubscriptionValidationRule(chargeOperationDto), chargeOperationDto.Id),
```

Comment:
Suggest renaming it like this, to me its' more to the point.

```suggestion
                CreateRuleContainer(new TaxIndicatorMustBeFalseForFeeValidationRule(chargeOperationDto), chargeOperationDto.Id),
                CreateRuleContainer(new TaxIndicatorMustBeFalseForSubscriptionValidationRule(chargeOperationDto), chargeOperationDto.Id),
```